diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/generation/Generation.kt

```diff
@@ -0,0 +1,72 @@
+package org.cqfn.diktat.ruleset.generation
+
+import com.squareup.kotlinpoet.FileSpec
+import com.squareup.kotlinpoet.KModifier
+import com.squareup.kotlinpoet.PropertySpec
+import com.squareup.kotlinpoet.TypeSpec
+import java.io.File
+import org.cqfn.diktat.ruleset.constants.Warnings
+import java.time.LocalDate
+
+private val AUTO_GENERATION_COMMENT =
```

Comment:
Please add in root `pom.xml` in `jacoco-maven-plugin` lines
```
                        <configuration>
                            <excludes>GenerationKt</excludes>
                        </configuration>
```
to not calculate test coverage of this file